### PR TITLE
Release 2.9.8

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Looking for the PR for the upcoming v2.9.8 production release? Well you've just found it, congratulations! :tada:

This is based on development.

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
This is a hotfix release to some sentry and user reported issues.
- [x] Verify that all feature flags are flipped appropriately
no changes
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
N/A

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.9.8": [
+      "[Fixed] Unicode emoji on Windows no longer render as monochrome outlines - #13914",
+      "[Fixed] App does not hang when discarding changes in some scenarios. - #13899",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] App no longer hangs when discarding changes in some scenarios - #13899",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+      "[Fixed] App no longer crashes sometimes when rebasing and cherry-picking. - #13889",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] App no longer crashes intermittently when rebasing and cherry-picking - #13889",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+      "[Fixed] Fix crash when attempting to move the app to the /Applications folder on macOS - #13886",
```
Comment (niik) [inline changelog.json:7]:
```suggestion
      "[Fixed] Fix crash when attempting to move the app to the /Applications folder on macOS - #13886",
      "[Fixed] App no longer crashes when checking for updates while the closing the window - #13892",
      "[Fixes] Restored application icon in \"Apps & Features\" on Windows - #13890",
```

**Review by niik — CHANGES_REQUESTED** [review]

**Review by billygriffin — COMMENTED** [review]
A couple suggestions!

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,14 @@
...
+      "[Fixed] App no longer crashes when checking for updates while the closing the window - #13892",
+      "[Fixes] Restored application icon in \"Apps & Features\" on Windows - #13890",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Restore application icon in \"Apps & Features\" on Windows - #13890",
```

**Review by sergiou87 — DISMISSED** [review]
🚀

**Review by billygriffin — APPROVED** [review]
yooooo!

**Review by niik — APPROVED** [review]

**Pull request merged**